app/Services/ImageGenerationService.php

```
@@ -4,120 +4,306 @@
+
+            // Create custom Browsershot instance if using AWS Lambda
+            $browsershotInstance = null;
+            if (config('app.puppeteer_mode') === 'sidecar-aws') {
+                $browsershotInstance = new BrowsershotLambda();
+            }
+
+            $browserStage = new BrowserStage($browsershotInstance);
+            $browserStage->html($markup);
+
+            // Set timezone from user or fall back to app timezone
+            $timezone = $user?->timezone ?? config('app.timezone');
```

PHPStan (2.1.33)

Using nullsafe property access "?->timezone" on left side of ?? is unnecessary. Use -> instead.


(nullsafe.neverNull)